life.bolls.bolls.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: life.bolls.bolls
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: bolls
+separate-locales: false
+tags: [proprietary]
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: bolls
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra bolls -t /app/bin
+      - install -Dm644 life.bolls.bolls.appdata.xml -t /app/share/metainfo
+      - install -Dm644 life.bolls.bolls.desktop -t /app/share/applications
+      - install -Dm644 life.bolls.bolls.svg -t /app/share/icons/hicolor/128x128/apps
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - unappimage bolls-0.1.1.AppImage
+          - rm bolls-0.1.1.AppImage
+          - mv squashfs-root bolls
+      - type: script
+        dest-filename: bolls
+        commands:
+          - zypak-wrapper /app/extra/bolls/bolls "$@"
+      - type: file
+        path: life.bolls.bolls.desktop
+      - type: file
+        path: life.bolls.bolls.svg
+      - type: file
+        path: life.bolls.bolls.appdata.xml
+      - type: extra-data
+        filename: bolls-0.1.1.AppImage
+        url: https://storage.googleapis.com/dbbba/bolls-0.1.1.AppImage
+        sha256: 8d379014e725bd839bd223ce2ad9989063b6b107f78025cdc4db2dffc784a50f
```

Comment:
You can't install it currently because the sha256 of the AppImage seems to be incorrect

Reply:
Sorry, but I do not know where to get that? I am doing it in the first time. How to know right sha256?